.tito/releasers.conf

```diff
@@ -1,6 +1,6 @@
 [fedora-git-all]
 releaser = tito.release.FedoraGitReleaser
-branches = rawhide f35 f34 epel9 epel8 epel7
+branches = epel7
```

Comment:
you've got this in reverse. This should go to mock-2 and what you have in mock-2 should go to main.

Reply:
Right, fixed.